js/table-toggle.js

```diff
@@ -0,0 +1,25 @@
+(function ($) {
+
+	'use strict';
+
+	$('.dropdown-content').on('click', 'a', function (e) {
+		e.preventDefault();
+
+		var table = '#tablepress-' + this.hash.replace('#', '');
+
+		$(table).show().siblings().hide();
+		$('.dropdown-content').hide();
+	});
+
+	$('.dropdown').on('mouseover', function () {
+		$(this).find('.dropdown-content').show();
+	});
+
+	$('.dropdown').on('mouseout', function () {
+		$(this).find('.dropdown-content').hide();
+	});
+
+
+	$('[href="#7"]').trigger('click');
```

Comment:
How are `#7` and `#16` links generated? I'd recommend putting a comment here to keep track of it over time.